Transfer/Sources/ScheduleTransferController.swift

```diff
@@ -266,13 +266,21 @@ extension ScheduleTransferController: ScheduleTransferView {
 
     func showConfirmation(handler: @escaping (() -> Void)) {
         let destinationData = presenter.sectionData[1] as? ScheduleTransferDestinationData
+        if destinationData?.transferMethod.type == HyperwalletTransferMethod.TransferMethodType.paperCheck.rawValue {
+            HyperwalletUtilViews.showAlert(self,
+                                           title: "mobileTransferSuccessMsg".localized(),
+                                           message: "",
+                                           actions: UIAlertAction.close({ (_) in
+                                            handler()
+                                           }))
```

Comment:
Can you attach a screenshot in Jira ticket, for this change? Thank You!

Reply:
Attached screenshot in Jira.